packages/front-end/components/Metrics/MetricsList.tsx

```diff
@@ -73,9 +73,18 @@ export function useCombinedMetrics({
 
   const combinedMetrics = [
     ...inlineMetrics.map((m) => {
-      const canDuplicate = permissionsUtil.canCreateMetric(m);
-      const canEdit = permissionsUtil.canUpdateMetric(m, {});
-      const canDelete = permissionsUtil.canDeleteMetric(m);
+      const canDuplicate =
+        m.managedBy === "admin"
+          ? permissionsUtil.canCreateOfficialResources(m)
+          : permissionsUtil.canCreateMetric(m);
+      const canEdit =
+        m.managedBy === "admin"
+          ? permissionsUtil.canUpdateOfficialResources(m, {})
+          : permissionsUtil.canUpdateMetric(m, {});
+      const canDelete =
+        m.managedBy === "admin"
+          ? permissionsUtil.canUpdateOfficialResources(m, {})
+          : permissionsUtil.canDeleteMetric(m);
```

Comment:
This has the same else/if issue as the back-end.  Also, there are additional checks below that skip rendering edit/delete actions if `managedBy` is not empty, regardless of what the permissions say.

I think it makes sense to move all of the logic here.  So instead of having `managedBy` checks during rendering, we can just do something like this:

```ts
let canEdit = permissionsUtil.canUpdateMetric(m, {});
let canDelete = permissionsUtil.canDeleteMetric(m);

// Additional check if managed by admins
if (m.managedBy === "admin" && !permissionsUtil.canUpdateOfficialResources(m, {}) {
  canEdit = false;
  canDelete = false;
}
// Disable entirely if managed by API or config.yml
else if (m.managedBy) {
  canEdit = false;
  canDelete = false;
}
```